clr/commands.py

```diff
@@ -129,6 +131,7 @@ class Namespace:
     longdescr: str
     command_specs: Dict[str, CommandSpec]
     command_callables: Dict[str, Callable]
+    instance: any
```

Comment:
`any` is a method (which takes an iterable, returning a boolean if any elements are true)

You probably want `typing.Any`.